CHANGELOG.md

```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [8.4.0] Stable Release
+### Added
+- Added support for Data Classification v2 [#1338](https://github.com/microsoft/mssql-jdbc/pull/1338) [#1373](https://github.com/microsoft/mssql-jdbc/pull/1373)
+- Added public API to return the database compatibility level [#1345](https://github.com/microsoft/mssql-jdbc/pull/1345)
```

Comment:
Added 'SQLServerDatabaseMetaData.getDatabaseCompatibilityLevel()' API to return the database compatibility level.